docs/reference/search/search-your-data/search-using-query-rules.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[search-using-query-rules]]
+== Searching with query rules
+++++
+<titleabbrev>Searching with query rules</titleabbrev>
+++++
+
+[[query-rules]]
+
+preview::[]
+
+_Query rules_ are rules that are defined inside {es} and are applied only to queries that match these rules.
+This allows more control over promoted results for specific queries only. For example, you may want to promote results only when users are searching for specific items.
+
+Rules are defined using the <<query-rules-apis,Query rules API>>.
+When defining a rule, we take into consideration the following:
+
+* The type of rule we want to apply.
```

Comment:
Could add quick examples here of what is meant by type, criteria, and actions. Otherwise, they don't mean anything until you start parsing the code examples below.

I was also initially confused because the rule has a type and the criteria has a type.